lib/Book/book_manager.rb

```diff
@@ -26,8 +33,9 @@ def add_book(publish_date, publisher, cover_state, title, color)
     new_book.label = Label.new(title, color)
     @labels.add_label(new_book, title, color)
     @books << new_book
-    puts "Book created successfully!
-    Publish date: #{new_book.publish_date}, Publisher: #{new_book.publisher}, Cover State: #{new_book.cover_state}"
+    puts "\e[35m 🎉 Book created successfully! 📖 ✨
+    \e[37m Publish date: #{new_book.publish_date}, Publisher: #{new_book.publisher}, Cover State: #{new_book.cover_state}"
+    puts "\e[34m"
```

Comment:
I noticed here that on starting the app and retrieving the books, this successful book creation message is displayed. 

To avoid this I think I should have created a separate method to print on the screen. 

This happens because of the implementation of the retrieve_books. I call the add book method to retrieve all the books.

Reply:
Hey @sunga12, This is because your store data branch was not merged into the `develop `branch when I created this branch That's why it didn't merge your changes in it but now I have merged the `develop `branch and it should solve the problem 😊
thanks for mentioning 😀

Reply:
yeah! you are right! I should put those messages inside `main.rb` method call. Thanks 😊